doc/api/util.md

```diff
@@ -2024,6 +2024,27 @@ util.types.isAnyArrayBuffer(new ArrayBuffer());  // Returns true
 util.types.isAnyArrayBuffer(new SharedArrayBuffer());  // Returns true
 ```
 
+### `util.types.isArrayBufferDetached(value)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `value` {any}
+* Returns: {boolean}
+
+Returns `true` if the value is a built-in [`ArrayBuffer`][] and
+is detached. Detached arrays have a byte length of 0, which prevents
+JavaScript from ever accessing underlying backing store.
+
+For example, we can end up with a detached buffer when using a BYOB
+(bring your own buffer) on a ReadableStream.
```

Comment:
```suggestion
is detached. For example, we can end up with a detached buffer
when using a BYOB (bring your own buffer) on a ReadableStream.
Detached arrays have a `byteLength` of 0, and their contents can
not be accessed in JavaScript.
```
The original text sounds a bit misleading: it's not length of 0 that prevents us from using detached buffers but detached state itself.

Reply:
Thank you for the review. I updated the pull request.